### PR TITLE
Refresh README

### PR DESCRIPTION
**Pull request by cocojoe**
Streamline doc
Fix links
Remove old information

### CODE REVIEW

**Comment by cocojoe** [comment]
@Nyholm I'm trying to tidy this up, I don't use symfony (well not for a long time) so appreciate any thoughts on this. Thx

File: README.md [hunk 1]
```diff
@@ -2,46 +2,31 @@
...
-for Symfony 2.x, stick to v1 branch
-
-##Demo
-
-Check the usage demo: https://github.com/auth0/jwt-auth-bundle/tree/master/example
+> For Symfony < 2.8 please see [v1](https://github.com/auth0/jwt-auth-bundle/tree/1.x.x-dev)
 
 ## Installation
 
-Check our docs page to get a complete guide on how to install it in an existing project or download a pre configured seedproject: https://auth0.com/docs/quickstart/backend/php-symfony/
```
Comment (glena) [inline README.md]: we should check why this quickstart is not online and link to the correct document

Comment (cocojoe) [inline README.md]: I'll reinstate it as it moved to https://auth0.com/docs/quickstart/backend/symfony
I missed it first time around

**Comment by Nyholm** [comment]
Sorry for being late to this PR. 
I think it all looks okey 👍

**Pull request merged**